.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,70 @@
+---
+name: Bug report
+about: Submit the behavior you consider invalid
+
+---
+
+<!--
+If you have all reproduction steps with a complete sample app, please share as many details as possible in the sections below.
+
+Make sure that you tried using the latest TestCafe version (https://github.com/DevExpress/testcafe/releases), where this behavior might have been already addressed.
+
+Before submitting an issue, please check CONTRIBUTING.md and existing issues in this repository (https://github.com/DevExpress/testcafe/issues) in case a similar issue exists or was already addressed. This may save your time (and ours).
+-->
+
+### What is your Test Scenario?
```

Comment:
Is this section mandatory? Can we make it optional (and say it here)?

Reply:
If you mean the "What is your Test Scenario?" section, it is not mandatory, but it was requested by  @AndreyBelym.

Reply:
okay, let it be this way

Reply:
I'm afraid if we make it optional, no one will bother to fill it. We really need this section, because it will help us in providing good workarounds.

Reply:
But do we need it in the proposal template?

Reply:
I think it might be useful for proposals too. Let's see how people will fill it when creating new Feature Requests. We can always remove it if it is redundant. For now, I would leave it (the Test Scenario section) in proposals too.